handler.js

```diff
@@ -2,11 +2,24 @@ export const hello = async (event, context) => {
   return {
     statusCode: 200,
     body: JSON.stringify({
-      message: `Go Serverless v1.0! ${(await message({ time: 1, copy: 'Your function executed successfully!'}))}`,
+      message: `Go Serverless v1.0! ${(await message({ time: 1, copy: 'Your function executed successfully!' }))}`,
     }),
   };
 };
 
+export const test1 = async (event, context) => {
+  return {
+    statusCode: 200,
+    headers: {
+      'Access-Control-Allow-Origin': '*'
+    },
```

Comment:
If this is needed going forward, should we also update `responses-lib.js` and add headers?

Reply:
done